cpp/pipelines/collect_journal_stats.cc

```diff
@@ -263,6 +264,31 @@ void UpdateDatabase(const IniFile &ini_file, const std::string &system_type, con
 }
 
 
+const std::string TEXT_FILE_DIRECTORY("/mnt/ZE020150/FID-Entwicklung/Zeder_Supervision");
+
+
+void UpdateTextFiles(const std::unordered_map<std::string, ZederIdAndPPNType> &ppns_to_zeder_ids_and_types_map,
+                     const std::unordered_map<std::string, DbEntry> &ppns_to_most_recent_entries_map)
+{
+    const auto DIRECTORY_PREFIX(TEXT_FILE_DIRECTORY + "/" + DnsUtil::GetHostname() + "/");
```

Comment:
I will merge it, but seems a bit dangerous, since `::gethostname()` used in `DnsUtil::GetHostname()` might return a wide variety of hostname versions depending on the configuration of the host system, so it might make sense to avoid the automatic creation of new directories and use a predefined manual setup instead.